installation/cmd/run.sh

```diff
@@ -117,10 +117,18 @@ fi
 
 if [[ ! ${SKIP_MINIKUBE_START} ]]; then
   echo "Provisioning Minikube cluster..."
-  if [[ ! ${DOCKER_DRIVER} ]]; then
-    kyma provision minikube --cpus ${MINIKUBE_CPUS} --memory ${MINIKUBE_MEMORY} --timeout ${MINIKUBE_TIMEOUT}
-  else
+  if [[ ! ${HYPERKIT} ]]; then
+    DOCKER_CPUS=$(docker info --format '{{json .}}' | jq -r '.NCPU')
+    DOCKER_MEMORY=$(($(docker info --format '{{json .}}' | jq -r '.MemTotal') / 1000000))
+    echo "Docker CPUS: " $DOCKER_CPUS
+    echo "Docker Memory: " $DOCKER_MEMORY
+    if [[ ("$DOCKER_CPUS" -lt "$MINIKUBE_CPUS") || ("$DOCKER_MEMORY" -lt "$MINIKUBE_MEMORY")]]; then
```

Comment:
This check is not stable because MINIKUBE_CPUS and MINIKUBE_MEMORY can be provided from outside via --minikube-cpus and --minikube-memory flags.